src/cryptonote_core/service_node_list.h

```diff
@@ -230,6 +230,7 @@ namespace service_nodes
     bool handle_uptime_proof        (cryptonote::NOTIFY_UPTIME_PROOF::request const &proof);
     void record_checkpoint_vote     (crypto::public_key const &pubkey, bool voted);
 
+    // TODO(loki): Remove rollback code once a majority of the network has upgraded, i.e. HF13
```

Comment:
I'm thinking it might actually not be possible to remove this until the first point release after the HF13 fork (because before the fork we need to remain compatible with everything all the way back to 4.0.3).